src/psyclone/tests/dynamo0p3_lma_test.py

```diff
@@ -884,6 +884,7 @@ def test_operator_bc_kernel_wrong_access_err(dist_mem):
 '''
 
 
+@pytest.mark.usefixtures("lfric_config")
```

Comment:
Again, the culprit was the top fixture scope. Fixed now.

Reply:
And since the module scope could easily allow future errors to be ignored by the CI because it always runs with the same number of nodes, I made the same change everywhere that was using module scope for setting up configuration instances, regardless of whether it currently modifies it or not.